experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -287,12 +291,22 @@ export class ModularChangeFamily
 		change: TaggedChange<ModularChangeset>,
 		isRollback: boolean,
 		repairStore?: ReadonlyRepairDataStore,
+		cache?: boolean,
```

Comment:
Document. Make sure to mention that the caching is based on the object identity of `change`, e.g. `change` must be `===` to get a cache hit.